configs/components/ruby.rb

```diff
@@ -3,6 +3,11 @@
   pkg.md5sum "091b62f0a9796a3c55de2a228a0e6ef3"
   pkg.url "http://buildsources.delivery.puppetlabs.net/ruby-#{pkg.get_version}.tar.gz"
 
+  if platform.is_windows?
+    pkg.add_source "http://buildsources.delivery.puppetlabs.net/windows/elevate/elevate.exe", sum: "bd81807a5c13da32dd2a7157f66fa55d"
```

Comment:
I was thinking about it, but it's a part of ruby in `puppet-win32-ruby`, so I figured this would be easiest. We definitely can make it it's own component though.